src/Dialogs/Task/AlternatesListDialog.cpp

```diff
@@ -124,7 +124,7 @@ AlternatesListWidget::Prepare(ContainerWindow &parent, const PixelRect &rc)
 void
 AlternatesListWidget::OnActivateItem(unsigned index)
 {
-  details_button->Click();
+  // details_button->Click();
 }
```

Comment:
This action will catch the is a key event when using the keyboard.
However by simply disabling it, clicking the list item with a pointing device is not possible anymore.

Clicking a list item should still open the details dialog.

Reply:
There's currently an issue with using the keyboard in the Alternates screen, in that you can only select the first or the last alternate in the list to navigate to using just the keyboard.

This is because there is no way to move to the Goto button from say the second or the third alternate in the list - I would expect the left / right arrow keys to do this, but instead they move the cursor to the first and last alternate in the list.

_(You can press the Enter key on an alternate to bring up the waypoint details, but then there's no Goto button in the waypoint details.)_


Am I right in assuming that your fix would allow the cursor to move between the Goto / Details / Close buttons and the alternates list using the arrow keys, from any position in the alternates list?